docs/rbd/deploy.md

```diff
@@ -73,6 +73,15 @@ make image-cephcsi
 | `stripeUnit`                                                                                        | no                   | stripe unit in bytes                                                                                                                                                                                                                                                                               |
 | `stripeCount`                                                                                       | no                   | objects to stripe over before looping                                                                                                                                                                                                                                                              |
 | `objectSize`                                                                                        | no                   | object size in bytes                                                                                                                                                                                                                                                                               |
+| `BaseReadIops`                                                                                      | no                   | the base limit of read operations per second |
+| `BaseWriteIops`                                                                                     | no                   | the base limit of write operations per second |
+| `BaseReadBytesPerSecond`                                                                            | no                   | the base limit of read bytes per second |
+| `BaseWriteBytesPerSecond`                                                                           | no                   | the base limit of write bytes per second |
+| `ReadIopsPerGB`                                                                                     | no                   | the limit of read operations per GiB |
```

Comment:
in the key `GB` is used and in the comments its mentioned `GiB` is that correct?